src/main/java/org/alliancegenome/curation_api/jobs/JobScheduler.java

```diff
@@ -238,7 +238,6 @@ public void cleanUpFileHistories() {
 		}
 	}
 
-	@Scheduled(cron = "0 0 0 ? * SUN")
```

Comment:
If not removing the method can you just comment this vs removing it.

Reply:
Sure.